### ISSUE TITLE
 a problem of training

### ISSUE THREAD

**Issue by chen-ming2019** [zhang-can/ECO-pytorch#36]
when I train ,I met a error called"Type error:'BatchNorm3d' object is not iterable" . I thought a lot about it ,but no outcome.
